.gitignore

```diff
@@ -9,3 +9,4 @@ paper.run.xml
 *out
 *.ipynb_checkpoints*
 .DS_Store
+tex/_minted-paper
```

Comment:
> paper.pyg
> paper.synctex.gz

Sure, with latexmk they're not being created for me though (but might as well throw them in!). 

Re the pdf file though, I think it's nice to have it rendered on github for quick access? Nice to have a particular readable version at a url as and when (use this a lot with other students etc...). Not terribly precious about it.